packages/loot-core/src/server/importers/ynab5.ts

```diff
@@ -162,6 +162,129 @@ async function importTransactions(
     entityIdMap.set(transaction.id, uuidv4());
   }
 
+  // Compute link between subtransaction transfers and orphaned transaction
+  // transfers. The goal is to match each transfer subtransaction to the related
+  // transfer transaction according to the accounts, date, amount and memo.
+  const orphanTransferMap = data.transactions
+    .filter(
+      transaction =>
+        transaction.transfer_account_id && !transaction.transfer_transaction_id,
+    )
+    .reduce((map, transaction) => {
+      const key =
+        transaction.account_id + '#' + transaction.transfer_account_id;
+      if (!map.has(key)) {
+        map.set(key, [transaction]);
+      } else {
+        map.get(key).push(transaction);
+      }
+      return map;
+    }, new Map<string, YNAB5.Transaction[]>());
+
+  const orphanSubtransfer = data.subtransactions.filter(
+    subtransaction => subtransaction.transfer_account_id,
```

Comment:
Is every subtransaction with a transfer_account_id orphaned?

Reply:
Unfortunately yes, subtransactions are missing `transfer_transaction_id`, and the corresponding transactions too.